android/build.gradle

```diff
@@ -91,11 +91,10 @@ android {
 
     if (useNextStorage) {
         testOptions {
-            unitTests.returnDefaultValues = true
-        }
-        compileOptions {
-            sourceCompatibility JavaVersion.VERSION_1_8
-            targetCompatibility JavaVersion.VERSION_1_8
+            unitTests {
+                returnDefaultValues = true
+                includeAndroidResources = true
+            }
         }
```

Comment:
Don't we need to keep this as long as `minSdkVersion` is below 24?

Reply:
Starting with AGP 4.2, it's 1.8 by default
https://developer.android.com/studio/releases/gradle-plugin#java-8-default